server/routes/content-fragments/additional-needs/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Router } from 'express'
+import { Services } from '../../../services'
+import retrieveAlnScreeners from '../../middleware/retrieveAlnScreeners'
+import retrieveConditions from '../../middleware/retrieveConditions'
+import retrieveStrengths from '../../middleware/retrieveStrengths'
+import retrieveChallenges from '../../middleware/retrieveChallenges'
+import retrieveSupportStrategies from '../../middleware/retrieveSupportStrategies'
+import AdditionalNeedsContentFragmentController from './additionalNeedsContentFragmentController'
+import asyncMiddleware from '../../../middleware/asyncMiddleware'
+
+const additionalNeedsContentFragmentRoutes = (services: Services): Router => {
+  const {
+    additionalLearningNeedsService,
+    challengeService,
+    conditionService,
+    strengthService,
+    supportStrategyService,
+  } = services
+  const router = Router({ mergeParams: true })
+
+  const additionalNeedsContentFragmentController = new AdditionalNeedsContentFragmentController()
+
+  router.get('/', [
```

Comment:
Starting to flesh out the route.
We are calling the same middlewares that the Overview and similar pages currently call; IE. a middleware for calling the APIs for each of screeners, support strategies, conditions etc etc
This is fine for now, but once we have a combined/composite API we can change these middlewares and just make 1 API call 👍